pkg/ingestor/parser/slsa/parser_slsa.go

```diff
@@ -21,13 +21,18 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/in-toto/in-toto-golang/in_toto"
+	scommon "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
+	slsa01 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1"
+	slsa02 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
+	slsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
```

Comment:
@mlieberman85 has an example we can use for testing in the spector repo: https://github.com/kusaridev/spector/blob/main/testdata/slsa_provenance_v1.json